AntonBabunin/src/main/java/ua/artcode/market/creator/TerminalCreator.java

```diff
@@ -1,24 +1,29 @@
 package ua.artcode.market.creator;
 
-
 import ua.artcode.market.bill.Bill;
 import ua.artcode.market.salesman.Salesman;
 import ua.artcode.market.terminal.Terminal;
 
 public class TerminalCreator {
 
     private static final int DEFAULT_SIZE = 20;
+    private static int countTerminal = 0;
```

Comment:
add terminal fields, something like `id` etc

Reply:
Already added in Terminal class

Reply:
if id already present why you use private static int countTerminal = 0